addons/binding/org.openhab.binding.sinope/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               lib/sinope-core.jar,\
```

Comment:
Yes, there are... I don't want to limit the use of the library to openhab only. If people want to protect their investment and use the core library standalone on a command, they can. At the same time, sinope-core library is a very low-level library. It is tied to the hardware gateway, which is only a socket to radio device. You can turn on logging and it will spit a lot a debug information.. but unless you have the api document, you won't be able to decode the payloads.

Reply:
Do you consider this library as stable in terms of there will only be change once every couple of years? The reason behind my question is that we do not want an update PR for thins binding every couple of month with a new binary jar file that has not been reviewed by anyone. Also in your repo https://github.com/chaton78/sinope-core you have a `license-EPL.txt` and I think the convention is to have a `license.txt`.

Also if you want to keep the code in your repository, may I ask what is the reason to create an openHAB binding if you do not want to contribute your code but rather create a kind of skeleton that points to a binary jar file?

Reply:
This library is stable for more than year. What I have implemented is enough for everyday use. It covers the same use-cases than their cloud neviweb (I don't like cloud-lockin). I consider the work done. I would only touch the binding to add new things. I think the binding term explain exactly what this is supposed to do. I see it as an adaptor pattern. I fit a low-level library into a defined framework. We would not have this discussion is Sinope Tech used upnp for example. 

My goals are simple:
I don't want to tie people to one home automation solution.
I don't want to duplicate code (because I want to keep the low-level API access).
If people want to contribute on the sinope-core library, add support for dimmers, low level thermostat, they can without having to fully implement an openhab binding. I'll do the work or they can... but at least they have the choice.

I don't mind to copy part of the code but not all the library code will fit in the openhab source tree (i.e standalone app to retrieve the API key). 
I am very open-minded: I can add also a -src.zip along with the lib. POM dep.? Let me know if you think about an integration pattern that would fit your goals.